lib/linguist/heuristics.rb

```diff
@@ -313,6 +313,16 @@ def call(data)
       end
     end
 
+    disambiguate ".props" do |data|
+      if /^(\s*)(<Project|<Import|<Property|<\/Project|<\/Import|<\/Property)/i.match(data) 
+        Language["XML"]
+      elsif /\w+\s*=\s*/i.match(data)
+        Language["INI"]
+      else
```

Comment:
Or should I just change the else case to `Text` until we found a better substitute?